tools/api_check/lib/apicheck.dart

```diff
@@ -35,7 +35,7 @@ List<String> getDartClassFields({
   final RegExp fieldExp = RegExp(r'_k(\w*)Index');
   final List<String> fields = <String>[];
   for (final CompilationUnitMember unitMember in result.unit.declarations) {
-    if (unitMember is ClassDeclaration && unitMember.name.lexeme == className) {
+    if ((unitMember is ClassDeclaration || unitMember is EnumDeclaration) && unitMember.name.lexeme == className) {
```

Comment:
Meh, looks like `unitMember` is not properly type promoted anymore, so `unitMember.name` doesn't resolve. I guess we'd have to do something like

```suggestion
    if ((unitMember is ClassDeclaration || unitMember is EnumDeclaration) && (unitMember as NamedCompilationUnitMember).name.lexeme == className) {
```

Reply:
uhhhhh, thanks!! I was still trying to see what the error is on the logs, and trying to get the engine to compile again here!

Reply:
Now it is something else. I'm going to cry. Testing here.